src/quacc/schemas/ase.py

```diff
@@ -271,6 +271,40 @@ def summarize_vib_and_thermo(
     )
 
 
+def summarize_neb_run(dyn: Optimizer) -> OptSchema:
+    """
+    Summarizes the NEB run results into an OptSchema.
+
```

Comment:
This function has several issues because it does not match how the other summarizers work. For instance, you'll see that the various summarizers return a `finalize_dict()` function where that function stores the results in a Maggma Store, among other things. Here, your results are never actually stored in your database. Please try to mimic what the other functions are doing.